RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -50,7 +51,7 @@ HLTPFDiJetCorrCheckerWithDiTau::HLTPFDiJetCorrCheckerWithDiTau(const edm::Parame
     : tauSrc_(consumes(iConfig.getParameter<edm::InputTag>("tauSrc"))),
       pfJetSrc_(consumes(iConfig.getParameter<edm::InputTag>("pfJetSrc"))),
       extraTauPtCut_(iConfig.getParameter<double>("extraTauPtCut")),
-      mjjMin_(iConfig.getParameter<double>("mjjMin")),
+      m2jjMin_(iConfig.getParameter<double>("mjjMin") * iConfig.getParameter<double>("mjjMin")),
```

Comment:
I think this needs to be improved.

A user might put `mjjMin = -1` in the PSet (with the intention of switching off the cut), and the plugin will 'silently' require `m2 > 1`.

One way to improve this is
```
m2jjMin_(std::max(0, iConfig.getParameter<double>("mjjMin")) * std::max(0, iConfig.getParameter<double>("mjjMin"))),
```
The same argument probably applies to other plugins touched by this PR.